### PR TITLE
Allow setting of the AWS region

### PR DESCRIPTION
**Pull request by alan-cooney**
Another small addition.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -433,6 +433,11 @@ module.exports = (params) => {
     AWS.config.update({ httpOptions: { agent: sslAgent } })
   }
 
+  // Update the AWS http agent with the region
+  if (typeof params.region !== false) {
+    AWS.congif.update({ region: params.region });
```
Comment (lupuszr) [inline index.js]: I presume it should be config :)

Comment (alan-cooney) [reply index.js]: Not sure how that made it through - I'll push the change

**Comment by alan-cooney** [comment]
Fixed the spelling mistake

File: index.js [hunk 2]
```diff
@@ -435,7 +435,7 @@ module.exports = (params) => {
 
   // Update the AWS http agent with the region
   if (typeof params.region !== false) {
```
Comment (jeremydaly) [inline index.js]: This should probably check for a `string` type, otherwise any type could be passed.

**Comment by alan-cooney** [comment]
Thanks @jeremydaly  - all sorted in the latest commit

**Comment by alan-cooney** [comment]
@jeremydaly are you happy with this now?

I've released a module which will depend on this feature - integrating with knex ( https://github.com/alan-cooney/knex-data-api-client ).

**Pull request merged**